### PR TITLE
Diff/changeHourCountdown

### PR DESCRIPTION
**Pull request by andreaSerranoBasetis**
- new magic line dates for each flavor in strings.xml
- refactor of: date variable, countdown ID's (days,min,seconds), countown layouts (3)

### CODE REVIEW

**Review by yvhalo — APPROVED** [review]

File: app/src/main/res/layout/layout_countdown_top.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
```
Comment (sharoncabrera) [inline app/src/main/res/layout/layout_countdown_top.xml]: RelativeLayout repeated @andreabtis

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <string name="OValues3">Respeto al medioambiente</string>
 
     <!--CountDown-->
-    <string name="cursa_date">24.02.2019, 10:00</string>
+    <string name="magicLineDate" translatable="false">24.02.2019, 00:00</string>
```
Comment (lotdrops) [inline app/src/main/res/values/strings.xml]: Pasar a long (unix time)

File: app/src/mallorca/res/values/strings.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="cityFlavor" translatable="false">Mallorca</string>
-
+    <string name="magicLineDate" translatable="false">07.04.2019, 00:00</string>
```
Comment (lotdrops) [inline app/src/mallorca/res/values/strings.xml]: Pasar a long (unix time)

File: app/src/valencia/res/values/strings.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="cityFlavor" translatable="false">Valencia</string>
-
+    <string name="magicLineDate" translatable="false">17.02.2019, 00:00</string>
```
Comment (lotdrops) [inline app/src/valencia/res/values/strings.xml]: Pasar a long (unix time)

File: app/src/main/res/layout/layout_countdown_top.xml (hunk 1 above)
Comment (lotdrops) [inline app/src/main/res/layout/layout_countdown_top.xml]: Remove outer relative layout

**Pull request merged**